### PR TITLE
DDF for Tradfri bulb gu10 w 400lm

### PR DESCRIPTION
**Pull request by dimitripb**
New DDF for Tradfri GU 10 W 400lm
#6709

### CODE REVIEW

**Comment by dimitripb** [comment]
@manup thanks for you comment on my other new DDF. Can you help me with this one? How can I rerun the failed check? There is nothing wrong with the json but the process timed out. Or should I close this PR and create a new one?

**Comment by dimitripb** [comment]
Check failed.

**Comment by manup** [comment]
Indeed strange, I think this is a problem with the GitHub Action not the PR itself 🤔

**Comment by ebaauw** [comment]
I doubt _SW Build ID_ is reportable.  I don't think I've ever seen a `cstring` attribute attribute that supports reporting.

File: devices/ikea/tradfri_bulb_gu10_w_400lm.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  ],
+  "bindings": [
+    {
+      "bind": "unicast",
+      "src.ep": 1,
+      "dst.ep": 1,
+      "cl": "0x0000",
+      "report": [
+        {
+          "at": "0x4000",
```
Comment (ebaauw) [inline devices/ikea/tradfri_bulb_gu10_w_400lm.json:78]: I doubt _SW Build ID_ is reportable.  I don't think I've ever seen a `cstring` attribute attribute that supports reporting.

Comment (manup) [reply devices/ikea/tradfri_bulb_gu10_w_400lm.json:78]: It's unusual but the Ikea devices do actually support it.

**Pull request closed without merging**